test/visual-regression/form-dynamic-validators.spec.ts

```diff
@@ -46,7 +46,7 @@ test("form dynamic inputs", async ({ page }) => {
   await page.getByTestId("fudis-button-9").click(); // remove required number validator
   await page.getByTestId("fudis-button-12").click(); // remove required date validator
   await page.getByTestId("fudis-button-15").click(); // remove required option validator
-  await page.getByText("Summer holidays").click(); // Check the first checkbox to remove error messages
+  await page.getByTestId("fudis-checkbox-group-1-item-1").check(); // Check the first checkbox to remove error messages
```

Comment:
'click()'

Reply:
In Playwright documentation `.check()` is used for checkboxes and radio-buttons: https://playwright.dev/docs/input#checkboxes-and-radio-buttons

But it also suggests using `.setChecked(true)`. If you test that, does it still fail locally for you?

Reply:
Actually it failed locally for me too now with `.check()` and passed with `.setChecked(true)` so I changed it.